lib/prelude.rb

```diff
@@ -3,6 +3,9 @@
 require_relative 'prelude/relation'
 require_relative 'prelude/enumerator'
 
+# Add support into AR::Base
+ActiveRecord::Base.include(Prelude::Preloadable)
```

Comment:
Rails considers this an anti-pattern because it'll load ActiveRecord::Base, and connect to the db I think, when prelude is loaded by bundler.

Generally you see libraries rely on:

```ruby
# First…
require "active_support" # Doesn't load the whole thing, just sets up autoloads.

# Then…
ActiveSupport.on_load :active_record do
  include Prelude::Preloadable # We're already ActiveRecord::Base here
  ActiveRecord::Relation.include Prelude::Relation
end
```

If that doesn't work we'd need a Railtie.

Reply:
Awesome, updated :)